library/kubevirt_ovm.py

```diff
@@ -60,7 +60,7 @@
         description:
             - "Name of a PersistentVolumeClaim existing in the same namespace
               to use as a base disk for the Offline VM."
-        required: true
+        required: false
```

Comment:
@tripledes I would keep it as `true*`
`* C(pvc) option is required to create the VM. It's not needed to remove the VM.`

And the same for ovm and vmrs.

Reply:
@alexxa it'd be inconsistent with `argument_spec`. If DOC says it's required, it should be required by both states, `present` and `absent`, IMHO.

Reply:
IMHO =) If a user see 'not required' for `present` and doesn't specif it, they will see a msg: "Either pvc or registrydisk is REQUIRED". Ok.. another option, what about `required" false`, and a note in bold: 

`**C(pvc) option is required to create the VM. It's not needed to remove the VM.**`